### PR TITLE
Add Dockerfile example that handles an arbitrary RUN instruction

### PR DESCRIPTION
**Pull request by alex1545**

### CODE REVIEW

File: testing/examples/run_instruction_arbitrary/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Ubuntu with bazel, gcloud and its dependencies preinstalled.
+
+FROM launcher.gcr.io/google/bazel:latest
+
+# Ensure new repos aren't compromised.
+COPY image_data/gcloud.pub.gpg /
+RUN apt-key add /gcloud.pub.gpg \
+    && rm /gcloud.pub.gpg
+
+# Add new repos to install bazel and google-cloud-sdk (including kubectl)
```
Comment (nlopezgi) [inline testing/examples/run_instruction_arbitrary/Dockerfile]: r/ bazel

File: testing/examples/run_instruction_arbitrary/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Ubuntu with bazel, gcloud and its dependencies preinstalled.
+
+FROM launcher.gcr.io/google/bazel:latest
+
+# Ensure new repos aren't compromised.
+COPY image_data/gcloud.pub.gpg /
```
Comment (nlopezgi) [inline testing/examples/run_instruction_arbitrary/Dockerfile]: Lets substitute installing google-cloud-sdk with apt get below to instead do a wget of
https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-220.0.0-linux-x86_64.tar.gz
and install using `/google-cloud-sdk/install.sh -q`

File: testing/examples/run_instruction_arbitrary/Dockerfile (hunk 1 above)
Comment (alex1545) [reply testing/examples/run_instruction_arbitrary/Dockerfile]: Done

File: testing/examples/run_instruction_arbitrary/Dockerfile (hunk 2 above)
Comment (alex1545) [reply testing/examples/run_instruction_arbitrary/Dockerfile]: Done

**Pull request merged**